code/processes/gateway.q

```diff
@@ -637,4 +647,17 @@ neg[h](`.gw.asyncexecjpt;(`.q.system;"sleep 10");enlist[`servertype]!enlist`rdb`
 h(`.gw.syncexec;"`$last .z.x";enlist[`tables]!enlist enlist`logmsgXXX)
 h(`.gw.syncexec;"`$last .z.x";`tables`servertype!(enlist`data;`rdb`hdb))
 
+\
```

Comment:
I guess this works, but I think it'd be better to just move the code further up the file? as a general pattern in TorQ (and other q code), once you hit a big block of comments with usage examples etc., you won't expect to find more actual code further down